java-client/session/src/main/java/io/deephaven/client/impl/Session.java

```diff
@@ -32,8 +32,8 @@ public interface Session
     // ----------------------------------------------------------
 
     /**
-     * Advanced usage, creates a new export ID for {@code this} session, but must be managed by the caller. Useful for
-     * more advanced integrations, particularly around doPut. Callers are responsible for {@link #release(ExportId)
+     * Advanced usage, creates a new table export ID for {@code this} session, but must be managed by the caller.Useful
+     * for more advanced integrations, particularly around doPut. Callers are responsible for {@link #release(ExportId)
```

Comment:
I can't comment on release(), but it should also have its comment edited, esp the "**only**"

Reply:
What's the suggestion for changing the release javadoc wrt "only"?

Reply:
We should introduce the proposed `newExportId(String type)`, document that, and have this one delegate to it with documentation that it's defaulting type to "table".

Reply:
I don't think introducing that would be of any use right now b/c there is no low-level way to register that export id w/ a server instance (save from the Table case with a doPut).

Reply:
I'm saying that this method is now documented to be suitable only for tables, that's the substantive change here.

The javadoc for `release()` says that it should "**only** be called in combination with exports returned from newExportId" (not my emphasis) - and since newExportId now says that it is only for tables, release is also only for tables.

Reply:
I think the documentation is still valid as-is; currently, `release()` is only meant for table export ids, but in the future if we add a way to get exports in another way, ie `newExportId(String type)`, then the language at that time can be changed to say "only use it with `newExportId()` or `newExportId(String type)`".

Reply:
I'm not sure I follow, @niloc132 ; Devin updated the JavaDoc for `newExportId()` to reflect the current reality, but `ExportId` and `release()` are not inherently Table specific.